src/envoy/BUILD

```diff
@@ -48,4 +48,5 @@ pkg_tar(
     mode = "0755",
     package_dir = "/usr/local/bin/",
     tags = ["manual"],
+    visibility = ["//visibility:public"],
```

Comment:
Hm, I was wondering why this wasn't needed for `//tools/deb:istio-proxy`, but it looks that there is a custom `pkg_tar` target there that packages Envoy binary in the same way.

Could you update `//tools/deb:debian-data` to use this instead and remove the old target?

Reply:
Sure. Done.